python/seldon_core/microservice.py

```diff
@@ -43,6 +46,25 @@
 GUNICORN_ACCESS_LOG_ENV = "GUNICORN_ACCESS_LOG"
 
 
+def grpc_health_check(self):
+    channel = grpc.insecure_channel(f"localhost:{os.environ.get(GRPC_SERVICE_PORT_ENV_NAME, DEFAULT_GRPC_PORT)}")
+    stub = prediction_pb2_grpc.ModelStub(channel)
+
+    batch = struct_pb2.ListValue()
+    data = prediction_pb2.DefaultData(ndarray=batch)
+    seldon_request = prediction_pb2.SeldonMessage(data=data)
+    stub.Predict(seldon_request, metadata=[('x-datadog-trace-id', '999')])
```

Comment:
I think we need to keep it constant, so health checks can be differed from regular requests

Reply:
👍🏻